man/mkdocs/mkdocs.yml

```diff
@@ -44,6 +44,17 @@ theme:
 # Customization
 extra:
   homepage: ./index.html
+  social:
+    - icon: /fontawesome/brands/discourse
+      link: https://discourse.osgeo.org/c/grass/62
+    - icon: fontawesome/brands/linkedin
+      link: https://linkedin.com/company/grass-gis
+    - icon: fontawesome/brands/twitter
```

Comment:
is this still correct (twitter -> X)?

Reply:
FYI:
https://lists.osgeo.org/pipermail/board/2025-February/013655.html